pkgs/tools/security/gnupg/20.nix

```diff
@@ -3,13 +3,13 @@
 
 # Each of the dependencies below are optional.
 # Gnupg can be built without them at the cost of reduced functionality.
-, pinentry ? null, x11Support ? true
+, pinentry ? null, guiSupport ? true
```

Comment:
I'm for this in theory, but against this in practice. The name `x11Support` is currently pretty much the standard in nixpkgs (with < 10 exceptions).